waiter/src/waiter/auth/authentication.clj

```diff
@@ -29,8 +29,14 @@
 
 (def ^:const AUTH-COOKIE-EXPIRES-AT "x-auth-expires-at")
 
+(def ^:const AUTH-COOKIE-EXPIRES-AT-ENTRY-PREFIX (str AUTH-COOKIE-EXPIRES-AT "="))
+
 (def ^:const AUTH-COOKIE-NAME "x-waiter-auth")
 
+(def ^:const AUTH-COOKIE-NAME-ENTRY-PREFIX (str AUTH-COOKIE-NAME "="))
+
+(def ^:const OIDC-CHALLENGE-COOKIE-PREFIX "x-waiter-oidc-challenge-")
```

Comment:
Unfortunately, we have to share the OIDC cookie name in this file but couldn't come up with a better alternative that would avoid additional computation.